src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -41,12 +41,7 @@ const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
       }
       // TODO: consolidate with the other save patron function
       // be aware the defaults are not necessarily the same in the different save patron functions
```

Comment:
This TODO mentions another save patron function. What function is that, and should we do a similar fix there?